src/engraving/libmscore/cmd.cpp

```diff
@@ -419,6 +419,9 @@ void Score::update(bool resetCmdState)
         ms->deletePostponed();
         if (cs.layoutRange()) {
             for (Score* s : ms->scoreList()) {
+                if (!s->isOpen() && ms->scoreList().size() > 1) {
```

Comment:
I'm wondering if it is possible to call layout only for the current score (which is visible on the screen at the moment)